src/snapred/backend/dao/ingredients/DiffractionCalibrationIngredients.py

```diff
@@ -14,6 +15,7 @@ class DiffractionCalibrationIngredients(BaseModel):
     runConfig: RunConfig
     instrumentState: InstrumentState
     focusGroup: FocusGroup
+    pixelGroup: PixelGroup
```

Comment:
it might be possible to remove the `instrumentState`.  Can you mark that as `TODO`?